### DISCUSSION TITLE
php run.php

### DISCUSSION THREAD

**Discussion by RamadhanP** [warifp/FacebookToolkit#77] [General]
Hallo mas, ijin bertanya saya ada kendala dibagian ini, jadi ketika klik php run.php engga muncul apa apa, kira kira solusinya gimana ya? terima kasih

**Comment by warifp** [comment]
Is there a screenshot?

**Reply by Theh-342** [reply]
@warifp 
![fac](https://github.com/warifp/FacebookToolkit/assets/155678262/e295a8d4-e287-4077-a735-4a31ae661114)

**Reply by Theh-342** [reply]
no output at all

**Comment by breebit8** [comment]
Apparently it no longer works or doesn't want to run, I enter the php run.php command and it doesn't load anything or does it open something wrong?

**Comment by breebit8** [comment]
Apparently it no longer works or doesn't want to run, I enter the php run.php command and it doesn't load anything or does it open something wrong?
